manual_process_work/vmain.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Top level simulation driver for use with verilator
+ */
+
+#include <verilated.h>
+#include <Vtop.h>
+
+vluint64_t main_time = 0;
+double sc_time_stamp() { return main_time; }
+
+int main(int argc, char *argv[]) {
+	Verilated::commandArgs(argc, argv);
+
+	Vtop *top = new Vtop;
+
+	for (; main_time < 1000 && !Verilated::gotFinish(); ++main_time) {
+		top->eval();
+	}
```

Comment:
Think this needs to increment time similar to what is on the current issue-2409-timing branch.